BackEndFlask/controller/Routes/Bulk_upload_routes.py

```diff
@@ -0,0 +1,42 @@
+from flask import jsonify, request, Response
+from controller import bp
+from flask_marshmallow import Marshmallow
+import pandas as pd
+import csv
+import json
+from Functions import genericImport
+from io import StringIO, BytesIO
+import os
+import shutil
+from controller.Route_response import create_bad_response, create_good_response
+
+@bp.route('/bulk_upload', methods = ['POST'])
+def upload_CSV():
+    try:
+        file = request.files['csv_file']
+        if not file:
+            return create_bad_response("Unsuccessfully uploaded a .csv file! No file selected.", "users", 400)
+        extension = os.path.splitext(file.filename)
+        if(extension[1]!= ".csv" and extension[1] != ".xlsx"):
+            return create_bad_response(f"Unsuccessfully uploaded a {extension[1]} file! Wrong Format", "users", 400)
+
+        if request.args.get("course_id"):
+            course_id = int(request.args.get("course_id"))
+            user_id = int(request.args.get("user_id"))
+            directory = os.path.join(os.getcwd(), "Test")
+            os.makedirs(directory, exist_ok=True)
```

Comment:
Personally, I would move this file handling stuff into a function called by the route, but it does not matter that much.